### ISSUE TITLE
To sort in an lexicographic order the objects in dict()

### ISSUE THREAD

**Issue by rzyu45** [smallbunnies/Solverz#78]
For example, I first run
```python
>>> from Solverz import iVar, Para, Ode
>>> x = iVar('x')
>>> y = iVar('y')
>>> lam = Para('lam')
>>> Ode('b', x + y * lam, diff_var=x).SYMBOLS
{'lam': lam, 'x': x, 'y': y}
```

Then I run, in a new python console,
```python
>>> from Solverz import iVar, Para, Ode
>>> x = iVar('x')
>>> y = iVar('y')
>>> lam = Para('lam')
>>> Ode('b', x + y * lam, diff_var=x).SYMBOLS
{'y': y, 'x': x, 'lam': lam}
```

The order of symbols in `Eqn.SYMBOLS` dict is not fixed. It is hard to write tests in such cases. Also, since the printing order of variables in the the generated modules is changing, we cannot reuse the compiled python functions if we reprint these modules.

I believe this should be attributed to sympy. To resolve this, we should use the `sort()` function to sort the items in `Eqn.SYMBOLS` in lexicographic order.

**Comment by rzyu45** [comment]
The locations of jac blocks in the Jac.blocks dict() vary in each run.

```python
m = Model()
m.x = Var('x', [1, 1])
m.c = Param('c', 2)
m.f1 = Eqn('f1', 1 + exp(m.x[0]) + sin(m.x[1]))
m.f2 = Eqn('f2', m.x[0] + m.c / 2 * m.x[1] ** 2)

F, y0 = m.create_instance()

current_file_path = os.path.abspath(__file__)
current_folder = os.path.dirname(current_file_path)

test_folder_path = current_folder + '\\Solverz_test_ae_module_printer_with_hvp'
sys.path.extend([test_folder_path])

pyprinter = module_printer(F,
                           y0,
                           'test_module',
                           jit=True,
                           directory=test_folder_path,
                           make_hvp=True)
pyprinter.render()

from test_module import mdl, y
from test_module.num_func import Hvp_, inner_Hvp, inner_Hvp0, inner_Hvp1, inner_Hvp2

assert inspect.getsource(Hvp_) == expected_Hvp_
assert inspect.getsource(inner_Hvp) == expected_inner_Hvp
assert inspect.getsource(inner_Hvp0.func_code) == expected_inner_Hvp0
assert inspect.getsource(inner_Hvp1.func_code) == expected_inner_Hvp1
assert inspect.getsource(inner_Hvp2.func_code) == expected_inner_Hvp2
```

In the above example, the `inner_Hvp0` and `inner_Hvp1` interchanges occasionally. The test cannot be passed. 
To resolve this, we ought to sort the jac blocks according to the keys. That is, first by the `SolVar` and then by the equation name.

**Comment by rzyu45** [comment]
Partially resolved in 9cf7ccffe8106b22dae9c61dc4c319c0c9423d34
